src/app/update-password.tsx

```diff
@@ -0,0 +1,97 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useNavigation } from 'expo-router';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { showMessage } from 'react-native-flash-message';
+import { KeyboardAvoidingView } from 'react-native-keyboard-controller';
+import { z } from 'zod';
+
+import { useUpdatePassword } from '@/api/auth/use-update-password';
+import { Button, ControlledInput, FocusAwareStatusBar, Text, View } from '@/ui';
+
+type FormValues = { password: string; passwordConfirmation: string };
+const SIX = 6;
+
+const schema = z
+  .object({
+    password: z
+      .string()
+      .min(SIX, 'Password must be at least 6 characters long'),
```

Comment:
`'Password must be at least ${SIX} characters long'` and maybe the SIX constant could be renamed to MIN_CHARS ?

Reply:
this should also be localized

Reply:
You're right! thanks